domaci22.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+<link rel="stylesheet" href="stil22.css"> 
+
+</head>
+
+<body>  <div class= "container">
+    
+    
+      <div class = "split">
+        <h1 class="grad"> Život u Splitu </h1>  
+        <p class="caremipsum"> Carem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor <br> incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud <br> 
+            exercitation ullamcu laboris nisi ut aliquip ex ea commode consequat. Aed do.
+        </p>
+        <br>
+        <button class="botun" id="zuta" type="button"  > Građani </button>
+        <button class="botun"  type="button"  > Edukacija i odgoj </button>
+        <button class="botun"  type="button"  > Stanovanje i gradnja </button>
+        <button class="botun"  type="button"  > Kultura </button> <br>
+        <button class="botun2"  id="zuta" type="button"  > Sport </button>
+        <button class="botun2"  type="button"  > Promet </button>
+        <button class="botun2"  type="button"  > Naknade i porezi </button>
+        <button class="botun2"  type="button"  > Gradski kotari i mjesni odbori </button>
+    </div>
+
+    <div class="drugi">
+    <h1 class="grad"> Službeni grad</h1>
+    <p class="caremipsum">Na mjernoj postaji smještenoj na području gradskog kotara Kamen s južne strane <br> Karepovca,
+        kontinuirano se provodemjerenja slijedećih parametara kvalitete zraka.
+    </p>
+      
+        <button class="botun" id="zuta" type="button"  > Gradski kotari i mjesni odbori </button>
+        <button class="botun"  type="button"  > Građani </button>
+        <button class="botun"  type="button"  > Edukacija i odgoj </button>
+        <button class="botun"  type="button"  > Sport </button>
+        <button class="botun2" id="zuta" type="button"  > Stanovanje i gradnja </button>
+        <button class="botun2"  type="button"  > Kultura </button>
+        <button class="botun2"  type="button"  > Promet </button>
+        <button class="botun2"  type="button"  > Naknade i porezi </button>
```

Comment:
Using digits may be tricky for some browsers, you might want to name `button-two` this one.